### ISSUE TITLE
Unable  to escape double quotes

### ISSUE THREAD

**Issue by miniscruff** [01mf02/jaq#30]
I am attempting to write a script that will update a "Parts" file with aws etags and part numbers. These etags are enclosed in quotes and I am getting an error when escaping and am wondering if this is a known issue or if there is a work around? ( I would take a look in the code but I don't know rust ).

Empty json file to start
```json
{ "Parts": [] }
```

Attempt to add a new item with escaped quotes
```sh
▶ jaq '.Parts += [{"ETag": "\"638650f144d2438b850c4530ad0249ce\"", "PartNumber": 1}]' parts.json
Error: Unexpected token, expected :, ,, (, -, ;, >, ), ], ., ?, <, !, {, =, %, 0, *, ", +, |, [, }, $, /
   ╭─[<unknown>:1:56]
   │
 1 │ .Parts += [{"ETag": "\"638650f144d2438b850c4530ad0249ce\"", "PartNumber": 1}]
   ·                                                        ┬
   ·                                                        ╰── Unexpected token \
───╯
Error: Unexpected token, expected //, <, -=, <=, }, +=, ?, *=, =, |=, +, ., ==, %=, ,, >, or, /, |, as, /=, %, !=, >=, *, -, [, and
   ╭─[<unknown>:1:24]
   │
 1 │ .Parts += [{"ETag": "\"638650f144d2438b850c4530ad0249ce\"", "PartNumber": 1}]
   ·                        ───┬──
   ·                           ╰──── Unexpected token 638650
───╯

```

**Comment by 01mf02** [comment]
Hello @miniscruff! You have indeed stumbled over a long-standing TODO in jaq, namely #1. I have just implemented this now in 4dbd85f22db936449cc8af88145992c5cf28902c, and your filter parses correctly with it. Have fun using jaq. ;)

**Comment by miniscruff** [comment]
oh sweet, I did do a search but did not find that issue
